appcircle_firebase_deploy/1.0.1/component.yaml

```diff
@@ -0,0 +1,48 @@
+platform: Common
+buildPlatform:
+displayName: Firebase Deployment
+description: "Deploy your web applications to Firebase Hosting"
+docsUrl: https://docs.appcircle.io/workflows/common-workflow-steps#firebase-deployment
+requiredComponents: "appcircle_flutter_web_build"
```

Comment:
@boztopuz I don't understand why this is required

Reply:
https://github.com/appcircleio/appcircle-firebase-deploy-component
In the readme file, "deploy your web application" sentence was written. Because I add it directly.